netbeans/libtools/nbproject/configurations.xml

```diff
@@ -40,7 +39,6 @@
       <itemPath>../../include/votca/tools/vec.h</itemPath>
       <itemPath>../../include/votca/tools/version.h</itemPath>
       <itemPath>../../include/votca/tools/votca_config.h.in</itemPath>
-      <itemPath>../../include/votca/tools/votca_gsl_boost_ublas_matrix_prod.h</itemPath>
```

Comment:
We are no longer using ublas because we switched to eigen right?

Reply:
yes